services/scheduler/clarifai_scheduler/main.py

```diff
@@ -0,0 +1,56 @@
+"""
+ClarifAI Scheduler Service
+
+This service runs periodic jobs including:
+- Concept hygiene
+- Vault synchronization
+- Reprocessing tasks
+"""
+
+import os
+import time
+import logging
+
+# Configure logging
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger(__name__)
+
+
+def main():
+    """Main entry point for the Scheduler service."""
+    logger.info("Starting ClarifAI Scheduler service...")
+    
+    # Check environment variables
+    vault_path = os.getenv('VAULT_PATH', '/vault')
+    automation_pause = os.getenv('AUTOMATION_PAUSE', 'false').lower() == 'true'
+    
+    logger.info(f"Vault path: {vault_path}")
+    logger.info(f"Automation paused: {automation_pause}")
+    
+    # Check scheduler configuration
+    concept_refresh_enabled = os.getenv('CONCEPT_EMBEDDING_REFRESH_ENABLED', 'true').lower() == 'true'
+    concept_refresh_cron = os.getenv('CONCEPT_EMBEDDING_REFRESH_CRON', '0 3 * * *')
+    
+    logger.info(f"Concept embedding refresh enabled: {concept_refresh_enabled}")
+    logger.info(f"Concept embedding refresh cron: {concept_refresh_cron}")
+    
+    # Main service loop (placeholder)
+    try:
+        while True:
```

Comment:
I think we only need to indicate the logger messages when there is a change from the previous state (ie, when first entering that state), not on every loop iteration